### PR TITLE
Docs: nuxt examples & Vuejs fix

### PR DESCRIPTION
**Pull request by StefanVDWeide**
I've added examples for deploy a Nuxt site both with server side rendering and as a static stite. Also, solved an issue with the VueJS docs where the wrong start command was written down.

### CODE REVIEW

**Comment by mezotv** [comment]
I dont think setting the port to 80 is required for this. Any port should work since it runs in a container anyways.

**Comment by StefanVDWeide** [comment]
> I dont think setting the port to 80 is required for this. Any port should work since it runs in a container anyways.

80 is the default port for NGINX which serves the files in this case. Setting it to anything else will result in a bad gateway error. I've tested 3000 and 81 just in case.

**Comment by mezotv** [comment]
> > I dont think setting the port to 80 is required for this. Any port should work since it runs in a container anyways.
> 
> 80 is the default port for NGINX which serves the files in this case. Setting it to anything else will result in a bad gateway error. I've tested 3000 and 81 just in case.

Oh so thats only an internal port then?

**Comment by StefanVDWeide** [comment]
> > > I dont think setting the port to 80 is required for this. Any port should work since it runs in a container anyways.
> > 
> > 
> > 80 is the default port for NGINX which serves the files in this case. Setting it to anything else will result in a bad gateway error. I've tested 3000 and 81 just in case.
> 
> Oh so thats only an internal port then?

Yeah exactly, since the container exposes port 80 the domain needs to be configured to point there

**Comment by Siumauricio** [comment]
I tested and both works, awesome

Can you change npm to pnpm? is more fast than npm

File: nuxt/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+2. **Add Environment Variables**:
+
+- Navigate to the "Environments" tab and add the following variable:
+
+   ```plaintext
+   NIXPACKS_BUILD_CMD="npm run generate" 
+   ```
+
+3. **Set the publish publish directory**:
```
Comment (Siumauricio) [inline nuxt/README.md:48]:
```suggestion
3. **Set the publish directory**:
```

**Review by Siumauricio — CHANGES_REQUESTED** [review]

**Comment by StefanVDWeide** [comment]
> I tested and both works, awesome
> 
> Can you change npm to pnpm? is more fast than npm

Great! Changed the example to `pnpm`

**Comment by mezotv** [comment]
> Great! Changed the example to `pnpm`

With pnpm you can drop the run so its just `pnpm generate`

**Comment by Siumauricio** [comment]
@StefanVDWeide I still see the package lock file, can you please run pnpm install to generate the pnpm lock file and delete the package.lock.json?

**Comment by StefanVDWeide** [comment]
> @StefanVDWeide I still see the package lock file, can you please run pnpm install to generate the pnpm lock file and delete the package.lock.json?

Ahh I see. I thought you meant to update the instructions. I've gone ahead and ran install with `pnpm` and remove the old lock file!

**Review by Siumauricio — APPROVED** [review]
Thanks!

**Comment by codeflorist** [comment]
This says `merged`, but there's no nuxt example in current repo?

**Pull request merged**